README.md

```diff
@@ -5,18 +5,19 @@
 ___
 ## Platform (core)
 
-[TRUNK](https://ci.openmrs.org/browse/TRUNK-MASTER) | [REST API](https://ci.openmrs.org/browse/RESTWS-RESTWS) | [FHIR API](https://ci.openmrs.org/browse/FHIR-FM2)
+[TRUNK](https://ci.openmrs.org/browse/TRUNK-MASTER) | [REST API](https://ci.openmrs.org/browse/RESTWS-RESTWS) | [FHIR API](https://ci.openmrs.org/browse/FHIR-FM2) | [sonaqube dashboard](https://ci.openmrs.org/browse/SON-OPENMRSCOREMASTER)
+
```

Comment:
why not capitalize the word `sonaqube dashboard` to suit the existing naming convention for the Plaform(core) budges?